src/ce/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@handsontable/react",
+  "version": "1.0.0-beta1",
+  "description": "Best Data Grid for React with Spreadsheet Look and Feel.",
+  "author": "Handsoncode <hello@handsoncode.net> (https://handsoncode.net)",
+  "homepage": "https://handsontable.com",
+  "license": "MIT",
+  "browser": "./dist/react-handsontable.js",
+  "main": "./commonjs/react-handsontable.js",
+  "module": "./es/react-handsontable.js",
+  "keywords": [
+    "handsontable",
+    "component",
+    "data",
+    "table",
+    "grid",
+    "data",
```

Comment:
"data" and "table" keywords are duplicated and "data table" is missing. Coincidence? 🤔